pkg/adapter/client.go

```diff
@@ -3,30 +3,46 @@
 package adapter
 
 import (
+	"fmt"
 	"os"
 
-	"github.com/sirupsen/logrus"
+	"github.com/pkg/errors"
 	"github.com/varlink/go/varlink"
 )
 
 // DefaultAddress is the default address of the varlink socket
 const DefaultAddress = "unix:/run/podman/io.podman"
 
+type VarlinkConnectionInfo struct {
+	RemoteUserName string
+	RemoteHost     string
+	VarlinkAddress string
+}
+
 // Connect provides a varlink connection
 func (r RemoteRuntime) Connect() (*varlink.Connection, error) {
-	var err error
-	var connection *varlink.Connection
-	if bridge := os.Getenv("PODMAN_VARLINK_BRIDGE"); bridge != "" {
-		logrus.Infof("Connecting with varlink bridge")
-		logrus.Debugf("%s", bridge)
+	var (
+		err        error
+		connection *varlink.Connection
+	)
+
+	// I'm leaving this here for now as a document of the birdge format.  It can be removed later once the bridge
+	// function is more flushed out.
+	//bridge := `ssh -T root@192.168.122.1 "/usr/bin/varlink -A '/usr/bin/podman varlink \$VARLINK_ADDRESS' bridge"`
```

Comment:
For debugging, I would Debugf() the bridge string as passed to NewBridge().

Reply:
the problem is that the remote-client has no debug capability. i was asked to turn it off

Reply:
Wait, why? We can't do logrus.Debugf?

Reply:
debug added back in